schemas/client.yml

```diff
@@ -87,6 +103,38 @@
         simply taking the last label will not work well for effective TLDs such as "co.uk".
       example: co.uk
 
+    - name: subdomain
+      level: extended
+      type: keyword
+      short: The subdomain of the domain.
+      description: >
+        The subdomain is all of the labels under the registered_domain in a partially qualified
+        domain name, and all of the labels except the hostname in a fully qualified domain name.
+        
+        If the domain has multiple levels of subdomain, such as "sub2.sub1.example.com",
+        the subdomain field should contain "sub2.sub1", with no trailing period.
+      example: east
+
+    - name: hostname
```

Comment:
What is this field's relationship to `client.subdomain`?

Reply:
I would consider it to be the first in a multi-level name in a p/fqdn 

Thinking about dns analysis, I might like to be able to analyze the host portion of a name (various features could be useful) as well as the [subdomains] as it stands (host + subdomains)

trying to avoid thinking about features etc., and just thinking about e.g. show me every domain name (requested or accessed) for a particular domain, and bucket by subdomain (assuming first name is a host designation - which even if its a subdomain with an a record, its still effectively a specific host...